### PR TITLE
H2 anchors stay in view

### PR DESCRIPTION
**Pull request by kagarmoe**
When you select menu sub-item in the left-nav, you'll land on the H2 for that item instead of 120px below it.

The fix only works when applied to the anchor element, not to the H2 element.

Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: www/source/stylesheets/_global.scss [hunk 1]
```diff
@@ -399,9 +399,17 @@ code {
...
+    content: "";
+    height: 220px;
+    margin: -220px 0 0;
+  }
+
 h2:not(:first-child) {
-  margin-top: rem-calc(40);
-}
+   margin-top: rem-calc(40);
+ }
```
Comment (davidMcneil) [inline www/source/stylesheets/_global.scss]: Looks like accidentally added whitespace?

File: www/source/stylesheets/_global.scss [hunk 2]
```diff
@@ -399,9 +399,17 @@ code {
...
 }
 
+h2
+  a::before {
+    display: block;
+    content: "";
+    height: 220px;
+    margin: -220px 0 0;
+  }
+
```
Comment (davidMcneil) [inline www/source/stylesheets/_global.scss]: Adding brackets around the `h2`'s contents would keep it consistent with the rest of the document. Adding a `visibility: hidden;` and `pointer-events: none;` could help indicate that this element is solely for layout purposes. Adding a comment about why this block is needed would also be helpful.

File: www/source/stylesheets/_global.scss (hunk 1 above)
Comment (kagarmoe) [reply www/source/stylesheets/_global.scss]: Thanks!

File: www/source/stylesheets/_global.scss (hunk 2 above)
Comment (kagarmoe) [reply www/source/stylesheets/_global.scss]: Thanks @davidMcneil! I tried using a height variable in the new code.

File: www/source/stylesheets/_code.scss [hunk 3]
```diff
@@ -44,4 +44,4 @@ pre {
 
     }
   }
-}
\ No newline at end of file
+}
```
Comment (kagarmoe) [inline www/source/stylesheets/_code.scss]: This was pulled in by an auto-fix when I closed the file.

**Pull request merged**